bigtable/bigtable-proxy/src/main/java/com/google/cloud/bigtable/examples/proxy/metrics/MetricsImpl.java

```diff
@@ -289,10 +311,30 @@ public void updateChannelCount(int delta) {
     channelCounter.add(delta);
   }
 
-  static Attributes unwrap(MetricsAttributes wrapped) {
+  private static Attributes unwrap(MetricsAttributes wrapped) {
     return ((MetricsAttributesImpl) wrapped).getAttributes();
   }
 
+  public static MetricData generateTestPresenceMeasurement(Resource resource) {
```

Comment:
Is this for testing? Add `@VisibileForTesting` annotation?

Reply:
nope, its used in Verify. I added a commentsto document the intent